test/e2e/flask/solana/common-solana.ts

```diff
@@ -7,31 +7,216 @@ import FixtureBuilder from '../../fixture-builder';
 import { ACCOUNT_TYPE } from '../../constants';
 import { loginWithBalanceValidation } from '../../page-objects/flows/login.flow';
 
-const SOLANA_URL_REGEX = /^https:\/\/.*\.solana.*/u;
-
+const SOLANA_URL_REGEX = /.*/u;
+// const SOLANA_RPC_PROVIDER = 'https://api.devnet.solana.com/';
+const SOLANA_PRICE_REGEX =
+  /^https:\/\/price-api\.metamask-institutional\.io\/v2\/chains\/solana:/u;
+const SOLANA_BITCOIN_MIN_API =
+  /^https:\/\/min-api\.cryptocompare\.com\/data\/pricemulti/u;
 export enum SendFlowPlaceHolders {
   AMOUNT = 'Enter amount to send',
   RECIPIENT = 'Enter receiving address',
   LOADING = 'Preparing transaction',
 }
 
+export const SOL_BALANCE = 50000000000;
+
+export const SOL_TO_USD_RATE = 225.88;
+
+export const USD_BALANCE = SOL_BALANCE * SOL_TO_USD_RATE;
+
+export const LAMPORTS_PER_SOL = 1_000_000_000;
+
```

Comment:
I'm seeing that all the RPC requests are mocked. Is there a plan to use a local network instead?
There are some npm packages for the solana-test-validator, which could be used ala ganache. This way, we won't need to mock balance changes, transactions etc.. and all these flows will be able to run out of the box relying on the local solana network.
Might be worth exploring (it can be done in a separate PR)

Reply:
yes, that's something that I have in mind already to investigate, but in a different PR